### PR TITLE
Updated the Factors control with the following: Highlights the selected level; updated the radio group to select only the selected level

### PR DESCRIPTION
**Pull request by deaspo**
Updated the Factors control with the following: Highlights the selected level; updated the radio group to select only the selected level @rdstern  @dannyparsons

### CODE REVIEW

File: instat/ucrFactor.vb [hunk 1]
```diff
@@ -80,14 +80,13 @@ Public Class ucrFactor
                     shtCurrSheet.ColumnHeaders(iSelectorColumnIndex).Text = strSelectorColumnName
                     InitialiseSelected()
                 Else
-                    shtCurrSheet.ColumnHeaders(iSelectorColumnIndex).DefaultCellBody = GetType(RadioButtonCell)
```
Comment (dannyparsons) [inline instat/ucrFactor.vb]: Don't we still need this line to set the Cell Type?

Comment (deaspo) [inline instat/ucrFactor.vb]: We do not need that line if we are using RadioButtonGroup

**Comment by dannyparsons** [comment]
The highlighting looks good. I just had one query about the Cell Type being set.

**Comment by deaspo** [comment]
Using the RadioButtonGroup is want we would want instead of setting the cellType as RadioButtonCell

File: instat/ucrFactor.vb (hunk 1 above)
Comment (dannyparsons) [inline instat/ucrFactor.vb]: Ok yes I see. As we are creating RadioButton Cells and then adding them to the column, it's not needed. 
If we change the dimension of the grid in another way, we may not have radio buttons, but I can't see us doing that at the moment so happy to not have this for now.

**Pull request merged**